components/nexus-orient/pom.xml

```diff
@@ -62,15 +62,9 @@
     </dependency>
 
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <optional>true</optional>
-    </dependency>
-
-    <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
-      <optional>true</optional>
+      <scope>test</scope>
     </dependency>
```

Comment:
There are some "Trial" test classes which refer to `SLF4JBridgeHandler`, but I think that might be down to copy+paste. Certainly they aren't run as part of the tests so I think the references to `SLF4JBridgeHandler.install` can be removed without disturbing anything else.